misterpasta.client/src/app/footer/footer.component.html

```diff
@@ -1 +1,33 @@
-<p>footer works!</p>
+<footer class="footer--container">
+  <div class="footer--item">
+    <p>
+      Heeft u een vraag voor ons? <br />
+      <a routerLink="/contact">Laat het zeker weten!</a>
+    </p>
+  </div>
+  <div class="footer--item">
+    <div class="footer--item__text">
+      <ul class="footer--references grid">
+        <li>
+          <a routerLink="/contact">Privacyverklaring</a>
+        </li>
+        <li>
+          <a routerLink="/contact">Contact</a>
+        </li>
+        <li>
+          <a routerLink="/about">Beoordelingen</a>
+        </li>
+        <li>
+          <a routerLink="/contact">Algemene voorwaarden</a>
+        </li>
+      </ul>
+    </div>
+  </div>
+  <div class="footer--item">
+    <div class="grid">
+      <p>Steurstraat 3d</p>
```

Comment:
Misschien een fictief adres gebruiken?

Reply:
Adres is veranderd naar een fictief adres.